helmfile/charts/kured-secret/templates/kured-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kured-secret
+type: Opaque
+stringData:
```

Comment:
Use `data:` instead 🙏🏻 ?

Reply:
I have now switched to data. But I see that most of your charts are using stringData :eyes:

Reply:
I like data because in stringData keys are not removed from the secret when applying and having the fields removed.